std/src/main/scala/algebra/std/char.scala

```diff
@@ -0,0 +1,19 @@
+package algebra
+package std
+
+package object char extends ByteInstances
```

Comment:
I'm guessing this was a copy/paste error. But that said, the law tests went through OK, which I think is due to us just import `algebra.std.all._` instead of individual imports. Not saying one way is better than the other, but frustrating that there isn't an easy way to verify both have all instances (without duplicating tests).